src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -296,7 +296,71 @@ asset::SAssetBundle CImageLoaderTGA::loadAsset(io::IReadFile* _file, const asset
     for (auto& img : images)
         img->drop();
     return SAssetBundle({core::smart_refctd_ptr<IAsset>(tex, core::dont_grab)});
+
+
+
 #else
+	core::vector<asset::ICPUImage*> images;
```

Comment:
you will only create ONE ICPUImage (previous system was fucked up thats why the API was changed).

The ICPUImage contains all mip-maps (but here you will only have one mip).

Reply:
you can delete this variable actually.